storage/api/Storage.Samples/GetPublicAccessPrevention.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_get_public_access_prevention]
+
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
+using System;
+
+public class GetPublicAccessPreventionSample
+{
+    public string GetPublicAccessPrevention(string bucketName = "your-unique-bucket-name")
+    {
+        // Gets Bucket Metadata and prints publicAccessPrevention value (either "unspecified" or "enforced").
+        var storage = StorageClient.Create();
+        var bucket = storage.GetBucket(bucketName);
+        var publicAccessPrevention = bucket.IamConfiguration.PublicAccessPrevention;
+
+        Console.WriteLine($"Public access prevention is {publicAccessPrevention} for {bucketName}.");
```

Comment:
Move this example to GetBucketMetadata sample instead:

https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/storage/api/Storage.Samples/GetBucketMetadata.cs

Instead of making a one off for Get Metadata publicAccessPrevention, update GetBucketMetadata sample.

Reply:
We purposefully added a separate region tag and sample for the docs for this sample. Having all the metadata printed is less clear to the reader.

Reply:
Also in dotnet in particular, the field is nested within IAM config so it's not even top level in the bucket metadata.

Reply:
We want to move away from a separate sample for maintainability reasons. I had a discussion with TW a while back about consolidating per sample gap work. 

https://docs.google.com/document/d/1hLBK8_k5V-GmGy_4BlV1JwnMhx1m73LhnZ0D_DSoyBg/edit#heading=h.k6extn85iq46

Reply:
Chatted with @tritone, I'll need to update process to reflect this change. Otherwise it will continue to be missed otherwise.